src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/ExternalReferencesTableNode.cs

```diff
@@ -102,6 +102,10 @@ public override ObjectData GetData(NodeFactory factory, bool relocsOnly = false)
                     // TODO: set low bit if the linkage of the symbol is IAT_PVALUE.
                     builder.EmitReloc(symbolAndDelta.Symbol, RelocType.IMAGE_REL_BASED_RELPTR32, symbolAndDelta.Delta);
                 }
+                else if (factory.Target.Abi == TargetAbi.CppCodegen)
```

Comment:
@MichalStrehovsky , should this be based off of TargetDetails rather than Cpp?

Reply:
Done. Thank you for suggestion!